src/view/com/composer/text-input/TextInput.web.tsx

```diff
@@ -240,6 +241,30 @@ export const TextInput = React.forwardRef(function TextInputImpl(
     }
   }, [onEmojiInserted])
 
+  // thats fix cmd + left navigates back when the editor is focused in a @mention
+  React.useEffect(() => {
+    if (editor) {
+      const handleKeyDown = (event: KeyboardEvent) => {
+        if ((event.metaKey || event.ctrlKey) && event.code === 'ArrowLeft') {
```

Comment:
in windows, the hotkey for navigating to the previous page on chrome is `alt + ←`, not `ctrl + ←`.

we should test if the bug is reproducible in windows and adjust the code accordingly.

Reply:
you're right! I don't have a windows machine nearby to test this, but the issue will very likely occur in that environment as well. i'll make the change.

Reply:
done